cumulusci.yml

```
@@ -392,6 +378,12 @@ flows:
     dev_org_namespaced:
         #description: Set up a namespaced scratch org as a development environment for unmanaged metadata
         steps:
+            1:
+                flow: beta_dependencies
+            2:
+                flow: deploy_ummanaged
```

```suggestion
                flow: deploy_unmanaged
```
Minor typo here causing this flow to fail. When I fixed it locally it ran fine.